src/Kafka/ConsumerGroups/components/OffsetDropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import { Dropdown, DropdownItem, DropdownToggle } from "@patternfly/react-core";
+import { FunctionComponent, SyntheticEvent, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { OffsetValue } from "../types";
+
+export type OffsetDropdownProps = {
+  value: OffsetValue;
+  onChange: (value: OffsetValue) => void;
+};
+
+export const OffsetDropdown: FunctionComponent<OffsetDropdownProps> = ({
+  onChange,
+  value,
+}) => {
+  const { t } = useTranslation("kafka");
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+
+  const onToggle = (isOpen: boolean) => {
+    setIsOpen(isOpen);
+  };
+
+  const offsetValueOption: { [key in OffsetValue]: string } = {
+    absolute: t("consumerGroup.offset.absolute"),
+    latest: t("consumerGroup.offset.latest"),
+    earliest: t("consumerGroup.offset.earliest"),
+    timestamp: t("consumerGroup.offset.timestamp"),
+  };
+
+  const onSelect = (
+    event?: SyntheticEvent<HTMLDivElement, Event> | undefined
+  ) => {
+    const eventTarget = event?.target as HTMLElement;
+    const value = eventTarget.innerText;
+    onChange(value as OffsetValue);
+    setIsOpen((prevState) => !prevState);
+  };
+
+  const dropDownItems = () => {
+    return Object.entries(offsetValueOption).map(([value, label]) => (
+      <DropdownItem key={value} value={value}>
+        {label}
+      </DropdownItem>
+    ));
+  };
+
+  return (
+    <Dropdown
+      onSelect={onSelect}
+      dropdownItems={dropDownItems()}
+      isOpen={isOpen}
+      id="offset-dropdown"
+      aria-label="offset-select-dropdown"
```

Comment:
This is not a proper label to present to a text reader. You should ask @jgiardino if in doubt about what to use here, but at the very least it should be the same label used for the wrapping `FormGroup`